classes/Http/Form/TalkForm.php

```diff
@@ -171,24 +145,4 @@ public function validateCategory(): bool
 
         return true;
     }
-
-    public function validateDesired(): bool
```

Comment:
See above comment about removing methods that suggest validation, but actually don't validate anything at all.